### DISCUSSION TITLE
Changing Tailwind Class Does Not Trigger HMR in Next13

### DISCUSSION THREAD

**Discussion by MattHeslington** [vercel/next.js#43321] [Help]
Hello,

I'm trying to use Next 13 with Tailwind, using a Javascript installation, not Typescript, but it's unusable at the moment due to the above problem .

```
npx create-next-app@latest --experimental-app
```

And following the instructions at [https://beta.nextjs.org/docs/styling/tailwind-css](https://beta.nextjs.org/docs/styling/tailwind-css) to implement Tailwind allows Tailwind to be used, though subsequent changes to classNames are not shown in the browser via HMR

The class is shown in the source code: (see bg-green-200)

![Screenshot 2022-11-24 at 16 42 21](https://user-images.githubusercontent.com/8208896/203735230-aac62740-677a-4445-8cb6-d131f9c3633a.jpg)

and the corresponding class is in the static css:

![Screenshot 2022-11-24 at 16 43 11](https://user-images.githubusercontent.com/8208896/203735444-3f3c7cf8-2e18-4acf-a6ae-817b9d8df2e2.jpg)

Can anyone see what's going wrong here, please?
Cheers,
Matt

**Comment by ephraimduncan** [comment]
Try deleting the `.next` directory and run the dev server again.

**Reply by MattHeslington** [reply]
Hi, 

I have tried that, it doesn't affect anything. I have raised the issue with the Tailwind on their repo: 

"Seems like the HMR from Webpack does not see that the global.css needs to be HMR-ed when the .tsx file is changed. Editing the global.css file directly works however. It seems like possibly a bug, but I'm afraid I'm not clued in enough with the Webpack or Next.js internals to give a more helpful answer."

**Comment by ellemedit** [comment]
Having a similar issue width WindiCSS. https://github.com/vercel/next.js/issues/44448
Only server component has the issue. I guess this is module cache issue.
